pyserini/dsearch/__main__.py

```diff
@@ -47,8 +47,15 @@ def define_dsearch_args(parser):
 def init_query_encoder(encoder, topics_name, device):
     encoded_queries = {
         'msmarco_passage_dev_subset': 'msmarco-passage-dev-subset-tct_colbert',
-        'nq_dev_dpr': 'nq-dev-dpr',
-        'nq_test_dpr': 'nq-test-dpr',
+        'nq_dev_dpr': 'nq-dev-dpr-multi',
+        'nq_test_dpr': 'nq-test-dpr-multi',
```

Comment:
move `dpr` to front so that they sort together?

Reply:
sure

Reply:
both topic name and encoded query name?